integration_tests/src/test/kotlin/integration/RunTimeoutIT.kt

```diff
@@ -0,0 +1,28 @@
+package integration
+
+import FlankCommand
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import run
+
+class RunTimeoutIT {
+
+    @Test
+    fun `cancel test run on timeout`() {
+        val name = this::class.java.simpleName
```

Comment:
and again

Reply:
https://github.com/Flank/flank/pull/1316#discussion_r522089515